ux.symfony.com/assets/styles/components/_Recipe.scss

```diff
@@ -0,0 +1,66 @@
+.Recipe {
```

Comment:
Ok for you if i create some generic "content page CSS base" ? I fear we have a lot of similar "component" like this one ?